cmake/modules/kconfig.cmake

```diff
@@ -128,32 +140,17 @@ else()
   set(_local_TOOLCHAIN_HAS_PICOLIBC n)
 endif()
 
-# A Kconfig.board indicates legacy style board definition, whereas
-# Kconfig.${BOARD} indicates new and improved style.
-if(EXISTS ${BOARD_DIR}/Kconfig.board AND NOT EXISTS ${BOARD_DIR}/Kconfig.${BOARD})
-  set(BOARD_SCHEME v1)
-elseif(NOT EXISTS ${BOARD_DIR}/Kconfig.board AND EXISTS ${BOARD_DIR}/Kconfig.${BOARD})
-  set(BOARD_SCHEME v2)
-elseif(EXISTS ${BOARD_DIR}/Kconfig.board AND EXISTS ${BOARD_DIR}/Kconfig.${BOARD})
-  message(WARNING "Mixed board scheme (v1 + v2) not allowed, using board scheme v2.")
```

Comment:
Some things are added in the first commit then removed or renamed in the second commit, would be ideal to have these all combined without adding then instantly removing/changing them

Reply:
Cleaned up and removed in original commit.